Force login in reset-credentials to federated users (#37214)
keycloak/keycloak: docs/documentation/server_admin/topics/login-settings/forgot-password.adoc

The text sent in the email is configurable. See link:{developerguide_link}[{developerguide_name}] for more information.

> When users click the email link, {project_name} asks them to update their password, and if they have set up an OTP generator, {project_name} asks them to reconfigure the OTP generator. For security reasons, the flow forces federated users to login again after the reset credentials and maintains internal DB users logged in if the same authentication session (same browser) is used. Depending on the security requirements of your organization, you may want to change both default behaviors.

Reviewer: ```suggestion
When users click the email link, {project_name} asks them to update their password, and if they have set up an OTP generator, {project_name} asks them to reconfigure the OTP generator. For security reasons, the flow forces federated users to login again after the reset credentials and keeps internal database users logged in if the same authentication session (same browser) is used. Depending on the security requirements of your organization, you can change the default behavior.
```
